vars/mtaBuild.groovy

```diff
@@ -24,6 +24,8 @@ import static com.sap.piper.Utils.downloadSettingsFromUrl
      * @possibleValues 'classic', 'cloudMbt'
      */
     'mtaBuildTool',
+    'dockerBasedModules',
```

Comment:
please add documentation comments for the new parameters, similar to the other parameters

Reply:
Sure.  If it's agreed that this is the proper approach.   Personally, I'd like the MTA file definition to be changed to include a way of specifying a module and not needing a path(as in the case of docker sourced modules).  This would require a new behavior in the MBT and/or the cf multiapps plugin.

See the discussion here.

[Deprecation of MTA Archive Builder ](https://github.com/SAP/jenkins-library/issues/1018)

Reply:
With regards to @katunin-ng this is planned for the future. So this could serve as a short-living workaround.